esdb-client/src/main/java/com/opencqrs/esdb/client/Marshaller.java

```diff
@@ -2,6 +2,7 @@
 package com.opencqrs.esdb.client;
 
 import com.opencqrs.esdb.client.eventql.ErrorHandler;
+import com.opencqrs.esdb.client.eventql.EventQLQuery;
```

Comment:
can someone explain to me why IDEA only links to the EventQLQuery in the JavaDoc when it is imported here, but it is working without the import in `QueryProcessingError`

Reply:
because they are in the same package?